src/app/features/section/subpages/section-details/section-details.component.ts

```diff
@@ -0,0 +1,72 @@
+import { AsyncPipe, JsonPipe, NgIf } from '@angular/common';
+import { ChangeDetectionStrategy, Component, inject, OnInit } from '@angular/core';
+import { MatButtonModule } from '@angular/material/button';
+import { MatCardModule } from '@angular/material/card';
+import { ActivatedRoute } from '@angular/router';
+import { selectLoggedUser } from '@core/store/user.selectors';
+import { Store } from '@ngrx/store';
+import { NavigationComponent, NavigationOption } from '@shared/ui';
+import { combineLatest, map } from 'rxjs';
+import { sectionDetailsActions } from '../../store/section.actions';
+import { selectSectionDetails } from '../../store/section.selectors';
+
+@Component({
+  selector: 'app-section-details',
+  standalone: true,
+  imports: [
+    NgIf,
+    AsyncPipe,
+    JsonPipe,
+    NavigationComponent,
+    MatButtonModule,
+    MatCardModule
+  ],
+  templateUrl: './section-details.component.html',
+  styleUrls: ['./section-details.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class SectionDetailsComponent implements OnInit {
+  private store = inject(Store);
+  private activeRoute = inject(ActivatedRoute);
+
+  sectionDetails$ = this.store.select(selectSectionDetails);
+  loggedInUser$ = this.store.select(selectLoggedUser);
+  navigationOptions!: NavigationOption[];
```

Comment:
jesli te navigationOptions są wymagane, to dodaj je w selektorze app-section-details[navigationOptions]

Reply:
ten komponent nie ma żadnych inputów więc nie widzę w tym sensu